domains/musaubrian.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "musaubrian",
+        "email": "musaubrian45@gmail.com"
+      },
+
+      "record": {
+        "CNAME": "musaubrian.netlify.app"
```

Comment:
Sorry, we don't support Netlify, but you can jave your domain redirect to your Netlify site using a `URL` record.

Reply:
Thanks for the feedback